src/api/spec/models/bs_request_action_spec.rb

```diff
@@ -1,13 +1,13 @@
 require 'rails_helper'
-# WARNING: If you change tests make sure you uncomment this line
-# and start a test backend. Some of the BsRequestAction methods
-# require real backend answers for projects/packages.
-# CONFIG['global_write_through'] = true
 
-RSpec.describe BsRequestAction, vcr: true do
+RSpec.describe BsRequestAction do
   let(:user) { create(:confirmed_user, login: 'request_user') }
 
-  context 'encoding of sourcediffs' do
+  before do
+    allow(User).to receive(:session!).and_return(user)
```

Comment:
:thought_balloon: I wonder why we mock this instead of setting `User.current`...

Reply:
well since `User.current=` is now private, we would have to start to do nasty things, which would probably lead to the question: "Why not just mock `session!` ?".. If you don't mind I would like to keep it like that.

Reply:
Sorry I've meant `User.session`, this is the right "replacement".